layout/loadGeoTerrain.m

```diff
@@ -0,0 +1,197 @@
+function [terrain, trajectories] = loadGeoTerrain(Monster, numUsers)
+	%% About
+	% Utility to load terrain information from shapefile files
+	% 
+	% The function loads a shapefile with roads, then drops a user in a random
+	% road. A walk is simulated along the road where the user has
+	% been dropped for up to the simulation time
+
+%% Load geo data
+
+seed = Monster.Runtime.seed;
+velocity = 1.5; % m/s
```

Comment:
Should this not be read from the Monster Config? We use it also for the fast-fading channel.